lib/src/internal/shard/shard.dart

```diff
@@ -255,17 +255,20 @@ class Shard implements IShard {
       case 1001:
         _reconnect();
         break;
+      case -1:
+        _connect(delay: 10);
```

Comment:
Maybe add a log message to indicate that the shard was disconnected because of no internet?

Reply:
it logs error directly from websocket already: `[2022-04-07 22:21:15.400327] [SEVERE] [Shard Manager] Shard 0 disconnected. Error: [SocketException: Failed host lookup: 'gateway.discord.gg' (OS Error: Temporary failure in name resolution, errno = -3)] Error code: [-1] | Error message: [SocketException]` It could be something different from that so I havent added anything else in case of logging